.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- Write BELOW The Headers and ABOVE The comments else it may not be viewable. -->
+<!-- You can view CONTRIBUTING.md for a detailed description of the pull request process. -->
+
+## About The Pull Request
+
+<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage senior developers from merging your PR! -->
+
+## Why This Is Good For ClanGen
+
+<!-- If this is a bugfix, you can remove this section. -->
+<!-- Please add a short description of why you think these changes would benefit the game. -->
+<!-- If this PR was the result of discussion/testing on the discord, please add a link to the discord conversation here. -->
+<!-- If you have multiple features that can stand on their own, or unrelated bugfixes, please create separate PRs for them. -->
+
+## Linked Issues
+
+<!-- If this is not a bugfix, you can remove this section. -->
```

Comment:
We do have some issues related to enhancements and not bugfixes, it would be good to adjust the wording here to account for it.  perhaps to: "If this is not related to any issues, you can remove this section.